packages/flutter_html_svg/.gitignore

```diff
@@ -29,6 +29,8 @@
 .pub-cache/
 .pub/
 build/
+coverage/
+scratch/
```

Comment:
Adding `coverage/` is fine, but its preferred that you ignore any files you add for personal use to your local `.git/info` configuration rather than the global `.gitignore`

Reply:
I will remove the `scratch/` directory from the PR.